src/services/recim.ts

```diff
@@ -16,13 +17,21 @@ type Activity = {
   Completed: boolean;
 };
 
+type Team = {
```

Comment:
Is this the way the ViewModel is represented coming from the database? We shouldn't use typing unless we know for sure what it looks like.

Reply:
<img width="175" alt="Screen Shot 2022-11-14 at 5 39 15 PM" src="https://user-images.githubusercontent.com/78386128/201783084-9c5eef40-5244-4f6d-8390-0f299c374063.png">
when creating a team (which is what you're doing right now) you will need to look at the ____UploadViewModels)
This is what your Team Type should look like